src/IB/IBFEInstrumentPanel.cpp

```diff
@@ -317,10 +317,11 @@ IBFEInstrumentPanel::IBFEInstrumentPanel(SAMRAI::tbox::Pointer<SAMRAI::tbox::Dat
     Utilities::recursiveMkdir(d_plot_directory_name);
 
     // set up file streams
-    if (SAMRAI_MPI::getRank() == 0)
+    static const int mpi_root = 0;
+    if (SAMRAI_MPI::getRank() == mpi_root)
     {
-        d_mean_pressure_stream.open(d_plot_directory_name + "/mean_pressure.dat");
-        d_flux_stream.open(d_plot_directory_name + "/flux.dat");
+        d_mean_pressure_stream.open(d_plot_directory_name + "/mean_pressure.dat", std::ofstream::out);
+        d_flux_stream.open(d_plot_directory_name + "/flux.dat", std::ofstream::out);
```

Comment:
I don't think this is right: this code will now open the streams here and then open them a second time (without closing) below.

I believe the way to fix this is to just get rid of these calls to open at this point.